web/src/components/storage/DeviceHeader.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) [2025] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+
+export type DeviceHeaderProps = {
+  title: string;
+  children: React.ReactNode;
+};
+
+export default function DeviceHeader({ title, children }: DeviceHeaderProps) {
+  const [txt1, txt2] = title.split("%s");
+
+  return (
+    <h4>
+      <span>{txt1}</span>
+      {children}
+      <span>{txt2}</span>
+    </h4>
```

Comment:
A comment for my self: check if going for a more PatternFly way does not break the layout here. I mean, 

```tsx
<Content component="h4">{txt} {children} {txt2}</Content>
```